### DISCUSSION TITLE
[RELEASE] Alleyway

### DISCUSSION THREAD

**Discussion by Paindeluxe** [nzp-team/nzportable#173] [Map Releases]
Greetings this is my Map that was inspired by the beginning Alleyway of Left for Dead's Mercy Hospital Campaign
This is a small map so it's good for simply killing zombies for a short while.

### [DOWNLOAD](https://github.com/nzp-team/nzportable/files/10611034/Alleyway.zip)

**Comment by MotoLegacy** [comment]
Hey there! This map isn't eligible for the "Meets Install Standard" label, you can check out the requirements and why it's useful on the post for it: https://github.com/nzp-team/nzportable/discussions/159

**Reply by Paindeluxe** [reply]
FIxed! Sorry 'bout that.

**Comment by Paindeluxe** [comment]
So many freaking edits that I spent more time on fixing these tiny issues than on making the map itself...

**Reply by Paindeluxe** [reply]
This is finally fully fixed (:

**Comment by MotoLegacy** [comment]
I have to remove this from Meets Install Standard because the map document text file is formatted incorrectly:
```
  + Downloading archive...
  + Decompressing archive...
  + Deleting archive...
  + Identified mapname as [ Alleyway ]
  + Found map information document
    - Name: [ Alleyway ]
    - Author: [ 1 ]
    - Description: [ 
        You don't know why your here...
        You don't know how you got here...
        You heard a horde of hungry zombies...
        You already know your doomed...
        Might aswell delay the inevitable...
        
        
        Pain deluxe
      ]
 ```
 
 Please consider fixing this, thanks!

**Reply by Paindeluxe** [reply]
Sure I‘ll do that, I just got a few questions. 

1. Do I have to reupload this Map to the Internet Archive?
2. Can you tag Models to not appear on PSP in recent Times? I haven’t fully checked out all the updates.
3. I‘d just have to look at what the other Maps did to figure out how it’s supposed to be, right?

**Reply by MotoLegacy** [reply]
1. To obey the spec yes, unfortunately I did not do a good job enforcing this in the past but now that I am working on an in-game downloader I must.
2. No, nor should it be necessary anyway, PSP performance is generally pretty great.
3. It's documented on the discussion page, with hyperlinks where necessary: https://github.com/nzp-team/nzportable/discussions/159

**Comment by PrimiseRichtofen** [comment]
Im currently highrounding right now bro

**Comment by irandiaz** [comment]
Buen map

**Comment by M0THRAK** [comment]
**Fun map!**
<img width="640" height="360" alt="image" src="https://github.com/user-attachments/assets/891728b0-aa62-4b3e-96cf-35ce591cbc54" />
